### ISSUE TITLE
"An error occurred"/"An unknown error occurred" when creating, saving, copying, or permanently deleting a page

### ISSUE THREAD

**Issue by FMCUSystemAdmins** [BookStackApp/BookStack#4648]
### Describe the Bug

We recently updated to v23.10 and have started getting the attached error when creating, saving, copying, or permanently deleting pages. For any of these actions, they actually do work as expected, we just get the error message also.

For the permanently delete action, we also notice that the count of pages in the Recycle Bin does not decrease after a page has been deleted. Unsure if related to this issue or separate.

### Steps to Reproduce

Creating page:

1. "New Page"
2. Make changes
3. Save
4. Error occurs
5. Page created as expected

Saving page:

1. "Edit"
2. Make changes
3. Save
4. Error occurs
5. Changes saved as expected

Copying page;

1. "Copy"
2. Select location
3. "Copy page"
4. Error occurs
5. Page copied as expected

Permanently deleting page:

1. Settings > Maintenance > Open Recycle Bin
2. Actions > Permanently Delete
3. "Confirm deletion"
4. Error occurs
5. **(Unsure if related to issue, only noticed while testing)** "Pages" count does not decrease
6. Page permanently deleted as expected

### Expected Behaviour

Creating, saving, copying, or permanently deleting a page should be successful without an error message.

### Screenshots or Additional Context

![image](https://github.com/BookStackApp/BookStack/assets/50454265/4508d486-a458-4a7c-8075-2fb3438b1f8d)

### Browser Details

Firefox (x64) v115.3.1esr

### Exact BookStack Version

23.10

**Comment by ssddanbrown** [comment]
Hi @FMCUSystemAdmins,
Did the `php artisan migrate` update step run without error? What do you see as output when attempting to re-run this command?

You can find out more detail for the error that's occurring by consulting the app error log:
https://www.bookstackapp.com/docs/admin/debugging/#error-log-file

**Comment by FMCUSystemAdmins** [comment]
This is the output of `php artisan migrate`:
![image](https://github.com/BookStackApp/BookStack/assets/50454265/0d776079-12e0-488e-af3d-40cd72c012ec)

Tried the page actions again and could not get the error to occur anymore. Could've swore we ran that command when we updated and there weren't any errors. Maybe we missed it. Will monitor future updates, but things seem to be fine now.

Thanks!
